### ISSUE TITLE
How to use --setFilteredGtToNocall and --maxNOCALLfraction with SelectVariants

### ISSUE THREAD

**Issue by chandrans** [broadinstitute/gatk#3143]
Make a note that when using SelectVariants with --setFilteredGtToNocall and --maxNOCALLfraction, you need to use them in two steps. First  --setFilteredGtToNocall then --maxNOCALLfraction.

----
User Report
----

Hello,

I have a VCF file on which I first use `VariantFiltration` (GATK v3.7) with various filter expressions, such as `"QD < 2.0"` and `"MQ < 40.0"`, as well as various genotype filter expressions, such as `"DP < 10"` and `"GQ < 20"`.

I then use `SelectVariants` with `--excludeFiltered` and `--excludeNonVariants`, but also with `--setFilteredGtToNocall`, `--maxFractionFilteredGenotypes 0.300000` and `--maxNOCALLfraction 0.300000`.

However, in the end, the output VCF file still contains a site with more than 30% missing genotypes.

To discard such a site, I have to run `SelectVariants` twice, the second time only with `--maxNOCALLfraction 0.300000`.

It hence looks like `--setFilteredGtToNocall` and `--maxNOCALLfraction` have to be used sequentially, in two different runs of `SelectVariants`. Am I right?

Best regards,
Timothée

This Issue was generated from your [forums] 
[forums]: http://gatkforums.broadinstitute.org/gatk/discussion/9795/how-to-use-setfilteredgttonocall-and-maxnocallfraction-with-selectvariants/p1

**Comment by vdauwera** [comment]
Is this a documentation request?

**Comment by chandrans** [comment]
Yes, basically to make a note that you need to use them in separate steps rather than use both in one go.
